charts/alfresco-search-service/templates/_helpers.tpl

```diff
@@ -16,6 +16,17 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Alfresco Search Service Host
+*/}}
+{{- define "alfresco-search.host" -}}
+{{- if index .Values.enabled -}}
```

Comment:
what is this top level enabled that I don't see in the values file?

Reply:
true that's a blind copy/paste from alfresco-common